themes/V3/Blank/style.less

```diff
@@ -167,6 +167,103 @@ body {
 	.watercolor11 { --wc : @watercolor11; }
 	.watercolor12 { --wc : @watercolor12; }
 
+	/* Image Masks */
+	[class*="imageMask"] {
+		position              : absolute;
+		height                : 200%;
+		width                 : 200%;
+		left                  : 50%;
+		bottom                : 50%;
+		--rotation            : 0;
+		--revealer            : none;
+		--checkerboard        : none;
+		--scaleX              : 1;
+		--scaleY              : 1;
+		-webkit-mask-image    : var(--wc), var(--revealer);
+		-webkit-mask-repeat   : repeat-x;
+		-webkit-mask-size     : 50%; //Scale only X to fit page width, leave height at aspect ratio, designed to hang off the edge
+		-webkit-mask-position : 50% calc(50% - var(--offset));
+		mask-image            : var(--wc);
+		mask-repeat           : repeat-x;
+		mask-size             : 50%;
+		mask-position         : 50% calc(50% - var(--offset));
```

Comment:
I'd suggest adjusting offset to include an `--offset-x` and `--offset-y` so it can be moved either direction

Reply:
Following my above comment, this *should* be tiled infinitely in the X direction (if I can figure out why it broke), so offset-x wouldn't do much except throw off the center of rotation. Offset lets you move the edge toward or away from the center, and rotate... rotates. Between those two you can already position the edge anywhere on the page.

Reply:
Gotcha that addresses the concern then!